share/spice/thumbtack/thumbtack.js

```diff
@@ -0,0 +1,47 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_thumbtack = function(api_result){
+
+        // Don't show anything if we weren't able to return 2 or more services.
+        // In future iterations, we should support a different view in the case
```

Comment:
@whalenrp we have built-in support for handling single items -- was that not working for you?

The UI would default to a map view with a box, using this template: https://github.com/duckduckgo/duckduckgo-answerbar-templates/blob/master/detail/places_detail.handlebars